kyaml/yaml/rnode.go

```diff
@@ -474,13 +474,32 @@ func (rn *RNode) ElementValues(key string) ([]string, error) {
 	return elements, nil
 }
 
+func (rn *RNode) ElementValuesList(keys []string) ([][]string, error) {
+	if err := ErrorIfInvalid(rn, yaml.SequenceNode); err != nil {
+		return nil, errors.Wrap(err)
+	}
+	elements := make([][]string, len(rn.Content()))
+
+	for i := 0; i < len(rn.Content()); i++ {
+		for _, key := range keys {
+			field := NewRNode(rn.Content()[i]).Field(key)
+			if !field.IsNilOrEmpty() {
```

Comment:
super nit: suggest letting the if statement match its content when possible. 

if field.IsNilOrEmpty() {
    // Attach nil or empty
} else {
    // Attach specific value.
}